<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatArea</class>
 <widget class="QWidget" name="ChatArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>543</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>496</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
	border-width: 1px;
    border-style: outset;
    border-radius: 4px;
    border-color: #000;
    background-color: #fff;
}

QPushButton {
    background-color: #ccc;
    border-style: outset;
    border-radius: 10px;
    border-color: #000;
}

QPushButton:hover {
    background-color: #fff;
}

QTextBrowser {
	border-width: 1px;
    border-style: outset;
    border-radius: 4px;
    border-color: #000;
    background-color: #fff;
}

QAbstractItemView {
	background-color: #fff;
}

QComboBox {
	background-color: #fff;
    selection-background-color: #fff;
}</string>
  </property>
  <widget class="QTextBrowser" name="usersOnlineDisplay">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>      emoticons:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="chatDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>301</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="sendMessage">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="emoticonDisplay">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>users online:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>username goes here</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
